types/coffeescript/package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "private": true,
+    "dependencies": {
+        "@babel/core": "^7.1.15"
```

Comment:
```suggestion
        "@babel/core": "~7.7.7"
```
https://github.com/jashkenas/coffeescript/blob/07f644c39223e016aceedd2cd71b5941579b5659/package.json#L43
which one is correct?

Reply:
The one in the `coffeescript` repository would correct, but will it work? `@types/babel__core` is at `7.1.16`

Reply:
Oh, so this one does not need this dependency. I just think @babel/core is outdated on DT (still at 7.1). 
Dependency would resolve to @types/babel__core